### ISSUE TITLE
[Bug]: In Edit taxon name, original combination disappears when relationship changed from species to subspecies

### ISSUE THREAD

**Issue by kandakoj** [SpeciesFileGroup/taxonworks#3067]
### Steps to reproduce the bug

```bash
In Edit taxon name, I am setting original combinations for a species that includes subspecies. 

When I set the original combination, I get the standard soft validation: relationship should move from species to subspecies, so I click fix.

The species entry changes to the nominate subspecies, but the combination disappears from the display below the fields.

I think it is just a display issue since as you can see in the search, the original combination
```

### Screenshot

When I set as current, everything displays correctly, I get expected soft validations
![TW_SetAsCurrent](https://user-images.githubusercontent.com/70152690/183091503-d2dacd26-6313-4b18-8ecd-24fc65c77bec.jpg)

Then I fix the soft validation about species/subspecies and the combination disappears from the display, although the data is still stored because it appears when I search for the species
![TW_FixSoftValidation](https://user-images.githubusercontent.com/70152690/183091785-8b60fb8f-dc9c-44a5-9a0c-8df8ab3113bf.jpg)
.

### Expected behavior

Original combination should display in the original combination and rank block.

### Environment

Production

### Version

0.27.3

### Browser Used

chrome

**Comment by mjy** [comment]
After I click the SV fix for nominotypical subspecies, then the full rendering does not display

<img width="834" alt="Screen Shot 2022-10-19 at 11 59 07 AM" src="https://user-images.githubusercontent.com/22600/196756893-e44bb77b-2560-4a5c-b197-5a93930d42bc.png">

**Comment by proceps** [comment]
Cached original combination is not set for the taxon.
![image](https://user-images.githubusercontent.com/5597748/196796669-c98639e9-725a-48eb-9716-f066b3a087cb.png)

**Comment by proceps** [comment]
No, sorry, was looking for a wrong taxon.... It is properly set for the one, I was experimenting with.
